frontend/javascripts/viewer/store.ts

```diff
@@ -621,14 +625,13 @@ export const combinedReducer = reduceReducers(
   VolumeTracingReducer,
   ProofreadingReducer,
   TaskReducer,
+  SaveReducer,
   FlycamReducer,
   FlycamInfoCacheReducer,
   ViewModeReducer,
   AnnotationReducer,
   UserReducer,
   UiReducer,
-  // SaveReducer needs to be behind Settings and Volumetracing reducer to react to changes SET MAPPING changes.
-  SaveReducer,
```

Comment:
ok, the comment is not true anymore, but somehow it was necessary to change it up? maybe add a comment that explains why that is necessary now?